venues/IEEE.org/2018/ITSC/python/create-invitations.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python
```

Comment:
this seems to be a create-review-invitations script?

Reply:
Yes, I'll rename it.